src/nvim/ex_getln.c

```diff
@@ -1082,6 +1082,7 @@ static int command_line_execute(VimState *state, int key)
     if (s->c == K_DOWN && ccline.cmdpos > 0
         && ccline.cmdbuff[ccline.cmdpos - 1] == '.') {
       s->c = (int)p_wc;
```

Comment:
Hmm, `Ctrl_Z` should always work in neovim without needing to fudge `KeyTyped`.

Reply:
Ok. I'll test in a separate commit.

Reply:
It doesn't work.